apps/chgrp.c

```diff
@@ -0,0 +1,26 @@
+/* vim: tabstop=4 shiftwidth=4 noexpandtab
+ * This file is part of ToaruOS and is released under the terms
+ * of the NCSA / University of Illinois License - see LICENSE.md
+ * Copyright (C) 2018 K. Lange
```

Comment:
You're not me!

Reply:
That's true, I just felt it might be misleading to change it considering this code is verbatim from your work on `chown.c` and the only thing I changed was what was passed into the function. 😅